autonomy/analyse/service.py

```diff
@@ -36,15 +43,100 @@
 from autonomy.configurations.loader import load_service_config
 
 
-REQUIRED_SETUP_PARAMETERES = (
-    "safe_contract_address",
-    "all_participants",
-)
+ABCI_CONNECTION_SCHEMA = {
+    "type": "object",
+    "properties": {
+        "config": {
+            "type": "object",
+            "required": [
+                "host",
+                "port",
+                "use_tendermint",
+            ],
+        },
+        "required": ["config"],
+    },
+}
 
-REQUIRED_PARAM_VALUES = (
-    "share_tm_config_on_startup",
-    "on_chain_service_id",
-)
+LEDGER_CONNECTION_SCHEMA = {
+    "type": "object",
+    "properties": {
+        "config": {
+            "type": "object",
+            "properties": {
+                "ledger_apis": {
+                    "type": "object",
+                    "patternProperties": {
+                        "^(ethereum|fetchai|cosmos)$": {
```

Comment:
That change makes it no better. Firstly, now we restrict to these 3. Secondly, these three have not the same params defined.

Instead, revert to ethereum only validation. But rather than fail, print a warning if non-ethereum configs are used.

Reply:
Addressed [771ce53](https://github.com/valory-xyz/open-autonomy/pull/1807/commits/771ce53807f351bca782686a1fa2d37cd4948d2e)